### DISCUSSION TITLE
What are the uses for `pam-any`?

### DISCUSSION THREAD

**Discussion by ChocolateLoverRaj** [ChocolateLoverRaj/pam-any#7] [General]
The only use I can think of is the `sudo` authentication.

**Comment by skruppy** [comment]
One use-case would be the use of a primary biometric factor to login like fingerprint (pam_fprintd) or face recognition (pam_howdy). But this factor not being available all the time since the needed webcam is not always available or the fingerprint reader too far away so you have to fallback to a secondary factor like a password. Also those biometric factors are not that reliant as a password. Hence you want to use both factors interchangeably, without waiting for a fingerprint reader to timeout or a password to be forced to be entered (the last one seems not to be possible without being hacky).
